src/applications/mhv-landing-page/tests/components/LandingPage.unit.spec.jsx

```diff
@@ -41,8 +41,8 @@ const setup = ({ initialState = stateFn(), props = {} } = {}) =>
 
 describe('LandingPage component', () => {
   it('renders', () => {
-    const { getByText } = setup();
-    getByText('My HealtheVet');
+    const { getByRole } = setup();
+    getByRole('heading', { level: 1, name: /My HealtheVet/ });
```

Comment:
Note: More specific role as it was also matching the `My HealtheVet` button in the sec nav